out/blocks/wd_admin_payment_main_form.tpl

```diff
@@ -141,26 +144,19 @@
           </select>
         [{/if}]
 
+        [{if $configField.type === 'button'}]
+          <input type="button" value="[{oxmultilang ident=$configField.multilangid}]" onclick="[{$configField.onclick}]" />
+        [{/if}]
+
         [{if $configField.description}]
           [{$oViewConf->getInputHelpHtml($configField.description)}]
         [{/if}]
       </td>
-      [{if $configKey === 'countryCode'}]
+      [{if $configField.validation}]
         <td>
-          <span id="country_code_error"></span>
+          <span id="[{$configField.validation}]"></span>
```

Comment:
Why not create this for every field, based on the ID? E.g.:

`<span id="[{$configKey}]_validation"></span>`

The element won't be visible unless it's used.

Reply:
👍